scripts/install

```diff
@@ -143,10 +143,11 @@ ynh_script_progression --message="Creating a PostgreSQL database..."
 
 db_name=$(ynh_sanitize_dbid --db_name=$app)
 db_user=$db_name
-db_pwd=$(ynh_app_setting_get --app=$app --key=psqlpwd)
+
 ynh_app_setting_set --app=$app --key=db_name --value=$db_name
 
 ynh_psql_test_if_first_run
+db_pwd=$(ynh_app_setting_get --app=$app --key=psqlpwd)
```

Comment:
This should be after `ynh_psql_setup_db`, and `ynh_psql_setup_db` should not pass `--db_pwd=$db_pwd`